### PR TITLE
Create password_and_2fa.md

### PR DESCRIPTION
**Pull request by EThom73**
**What?**
Made Tech Password and 2FA policy

**Why?**
ISO 27001 compliance

### CODE REVIEW

File: ims/company/password_and_2fa.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- You must not use the same password for Made Tech access as for non-Made Tech/personal access to services or devices.
+
+## Password Management
+We use password management for Made Tech services. The password manager is maintained by the IMS management group. Individual vaults are managed by the teams who use them.
+
+Your password to access your password manager should confirm to the password policy. Biometrics is also an acceptable method of accessing password managers.
+
+If you have reason to believe your password manager password has been compromised, report it immediately to the IMS manager or CSO. 
+
+## Obtaing Passwords via Social Engineering
```
Comment (lukemorton) [inline ims/company/password_and_2fa.md]: Typo "Obtaing", sounds like an orange flavoured fizzy drink.

File: ims/company/password_and_2fa.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Cracking passwords or a brute force attack are not the only ways in which individuals with bad intentions can get into the Made Tech systems. Social engineering also makes us vulnerable. Attackers will use techniques to gain your trust, or take advantage of inattention or a desire to be helpful to obtain passwords. Keep in mind the following:
+
+- Made Tech passwords are shared on a need-to-know basis. All passwords are to be treated as sensitive, internal information belonging to Made Tech
+- Only share Made Tech passwords with persons / systems authorised by the system owner, and only share by secure means
+- Passwords must not be revealed over the phone
+- Do not write passwords down and store them anywhere in the office. 
+- Do not store passwords in a file on a computer system or mobile device (phone, tablet) without encryption.
+- Do not reveal a password on forms
+- Do not hint at the format of a password ("My hometown/name/etc)
```
Comment (lukemorton) [inline ims/company/password_and_2fa.md]: Is this also a typo?

File: ims/company/password_and_2fa.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- Be sceptical of unusual requests
+- Be aware of potential phishing attacks
+- Consider who may be shoulder surfing
+- Hover your mouse over links in e-mail to ensure the web address makes sense
+- Verify the identity of a requester before sharing information
+- When in doubt, do not respond!
+
+It's always better to be safe than sorry. Always check with the IMS Manager or CSO is you think something is amiss.
```
Comment (lukemorton) [inline ims/company/password_and_2fa.md]: We should explain acronyms, or link to definitions.

File: ims/company/password_and_2fa.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+## 2FA
+Where a service provides the ability to authenticate via 2FA, 2FA must be used. 
+
+# Related Standards, Policies and Processes
+ - [AUP Policy](link)
```
Comment (lukemorton) [inline ims/company/password_and_2fa.md]: Can we just use the full name here, rather than an acronym?

File: ims/company/password_and_2fa.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# Password and 2FA policy
```
Comment (lukemorton) [inline ims/company/password_and_2fa.md]: Can we just use the full name here, rather than an acronym?

File: ims/company/password_and_2fa.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+## 2FA
+Where a service provides the ability to authenticate via 2FA, 2FA must be used. 
+
+# Related Standards, Policies and Processes
+ - [AUP Policy](link)
+ - [Lost or Stolen policy](link)
+ - [How to report an incident](link)
+ - [Device Security](link)
+ - [BYOD Policy](link)
```
Comment (lukemorton) [inline ims/company/password_and_2fa.md]: Can we just use the full name here, rather than an acronym?

File: ims/company/password_and_2fa.md (hunk 3 above)
Comment (EThom73) [reply ims/company/password_and_2fa.md]: These are explained on the Security README.
https://github.com/madetech/handbook/tree/master/ims#what-is-it

Comment (EThom73) [reply ims/company/password_and_2fa.md]: Also CSO/IMS Manager seems to be (based on documentation I have read from other companies implementing this) to be standard acronyms for Security related roles.

File: ims/company/password_and_2fa.md (hunk 6 above)
Comment (EThom73) [reply ims/company/password_and_2fa.md]: BYOD was already in use on the old handbook. Well established precedent for the acronym 😉

File: ims/company/password_and_2fa.md (hunk 5 above)
Comment (EThom73) [reply ims/company/password_and_2fa.md]: 2FA is a standard cybersecurity related acronym. However, for clarity I have expanded the acronym in the first line of P1 below

File: ims/company/password_and_2fa.md (hunk 3 above)
Comment (EThom73) [reply ims/company/password_and_2fa.md]: As an update there will be a full IMS Team section to the guide with acronyms fully explained.

**Pull request merged**